### PR TITLE
sysvar: add 2 sysvars

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11328 https://github.com/pingcap/docs-cn/pull/11375 https://github.com/pingcap/docs-cn/pull/11431 https://github.com/pingcap/docs-cn/pull/11505
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2955,14 +2964,22 @@ explain select * from t where age=5;
...
-### `tidb_shard_allocate_step` <span class="version-mark">New in v5.0</span>
+### tidb_shard_allocate_step <span class="version-mark">New in v5.0</span>
 
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Default value: `9223372036854775807`
 - Range: `[1, 9223372036854775807]`
 - This variable controls the maximum number of continuous IDs to be allocated for the [`AUTO_RANDOM`](/auto-random.md) or [`SHARD_ROW_ID_BITS`](/shard-row-id-bits.md) attribute. Generally, `AUTO_RANDOM` IDs or the `SHARD_ROW_ID_BITS` annotated row IDs are incremental and continuous in one transaction. You can use this variable to solve the hotspot issue in large transaction scenarios.
 
+### tidb_simplified_metrics
```
Comment (TomShawn) [inline system-variables.md:2975]: not in v6.2

File: system-variables.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ mysql> SELECT * FROM t1;
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Default value: `utf8mb4_bin`
-- The default collation for the server.
+- The default collation for a database when the database is created.
```
Comment (shichun-0415) [inline system-variables.md:169]: Need update according to the [latest Chinese](https://github.com/pingcap/docs-cn/pull/11505/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R150)

File: system-variables.md [hunk 3]
```diff
@@ -268,7 +268,7 @@ For more possible values of this variable, see [Authentication plugin status](/s
 
 ### identity
```
Comment (shichun-0415) [inline system-variables.md:269]:
```suggestion
### identity <span class="version-mark">New in v5.3.0</span>
```

File: system-variables.md (hunk 2 above)
Comment (TomShawn) [reply system-variables.md:169]: It is according to the latest Chinese: 该变量表示创建数据库时默认的排序规则。

Comment (TomShawn) [reply system-variables.md:169]:
```suggestion
- The default collation used when the database is created.
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**